guides/source/upgrading_ruby_on_rails.md

```diff
@@ -102,6 +102,18 @@ Upgrading from Rails 7.0 to Rails 7.1
 
 For more information on changes made to Rails 7.1 please see the [release notes](7_1_release_notes.html).
 
+### Development and test environments secret_key_base file changed
+
+In development and test environments, the file from which Rails reads the `secret_key_base` has been renamed from `tmp/development_secret.txt` to `tmp/local_secret.txt`.
+
+You can simply rename the previous file to `local_secret.txt` to continue using the same secret, or copy the key from the previous file to the new one.
+
+Failure to do so will cause Rails to generate a new secret key in the new file `tmp/local_secret.txt` when the app loads. 
+
+This will invalidate all existing sessions/cookies in development and test environments, and also cause other signatures derived from `secret_key_base` to break, such as ActiveStorage/ActionText attachments.
```

Comment:
```suggestion
This will invalidate all existing sessions/cookies in development and test environments, and also cause other signatures derived from `secret_key_base` to break, such as Active Storage/Action Text attachments.
```